rules/macos/credential_access_suspicious_web_browser_sensitive_file_access.toml

```diff
@@ -3,20 +3,20 @@ creation_date = "2020/01/04"
 integration = ["endpoint"]
 maturity = "production"
 min_stack_comments = "New fields added: required_fields, related_integrations, setup"
-min_stack_version = "8.3.0"
-updated_date = "2023/11/15"
+min_stack_version = "8.11.0"
+updated_date = "2024/02/06"
 
 [rule]
 author = ["Elastic"]
 description = """
-Identifies the execution of a process with arguments pointing to known browser files that store passwords and cookies.
+Identifies the access or file open of web browser sensitive files by an untrusted/unsigned process or osascript.
 Adversaries may acquire credentials from web browsers by reading files specific to the target browser.
 """
 from = "now-9m"
 index = ["logs-endpoint.events.*"]
```

Comment:
Since these are only file events, we can scope index tighter to `logs-endpoint.events.file*` to improve performance.